mlflow/fastai.py

```diff
@@ -380,7 +380,9 @@ def load_model(model_uri):
 
 @experimental
 @autologging_integration(FLAVOR_NAME)
-def autolog(log_models=True, disable=False, exclusive=False):  # pylint: disable=unused-argument
+def autolog(
+    log_models=True, disable=False, exclusive=False, disable_for_untested_versions=False
```

Comment:
As part of the documentation effort here, can we document the min and max tested versions for each autologging integration? We should be able to accomplish this by augmenting the `@autologging_integration` decorator to add / format some content into the docstring (see the definition of `@experimental` in `annotations.py` - https://github.com/mlflow/mlflow/blob/2d3ac48b01da021d220fb1bf4450c70976f3b5d7/mlflow/utils/annotations.py#L5-L17 - for a similar reference).

One thought would be to prepend a notice like the following before the remainder of the docstring:

```
Autologging for fastai has been tested with versions ``1.0.60`` thru ``1.0.61`` and may not work for versions outside of this range. To try autologging with untested versions, specify ``disable_for_untested_versions=True``.
```